assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -1194,7 +1195,7 @@ export default Component.extend({
         user: draft.replyToMsg.user,
       };
     }
-    this.chat.setDraftForChannel(this.chatChannel, draft);
+    this.chatDraftHandler.setDraftForChannel(this.chatChannel, draft);
```

Comment:
Why do we need to pass the draft here when it's supposed to be hold in the handler?